src/components/nav.js

```diff
@@ -16,17 +16,22 @@ const Nav = () => {
     },
   ];
   return (
-    <nav>
-      <ul>
-        {links.map((link) => (
-          <li key={link.id}>
-            <NavLink to={link.path} exact>
-              {link.text}
-              {' '}
-            </NavLink>
-          </li>
-        ))}
-      </ul>
+    <nav className="navbar">
+      <div className="nav1">
+        <header>
+          <h1 className="Title">Bookstore CMS</h1>
+        </header>
+        <ul className="Links">
+          {links.map((link) => (
+            <li key={link.id} className={link.name}>
+              <NavLink to={link.path} exact>
+                {link.text}
+                {' '}
+              </NavLink>
+            </li>
+          ))}
+        </ul>
+      </div>
     </nav>
```

Comment:
- [ ] Your header is missing the avatar icon, kindly add it so it matches the design.
![image](https://user-images.githubusercontent.com/33767951/140624335-7648ceec-a996-4c4c-b3bf-07e1a3b7f6d7.png)